proto/streamlit/proto/FileReader.proto

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright 2018-2019 Streamlit Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+syntax = "proto3";
+
+message FileReader {
```

Comment:
Add comments describing what "id" and "file" are. (Looks like "file" is the _name_ of the file, and not the file itself? If so, then "filename" might be better?)

Reply:
Is not the file name, is the el fileID devuelto por el file_uploader. Pero puedo cambiarlo :)